packages/mgt-components/src/components/mgt-tasks/mgt-tasks.graph.planner.ts

```diff
@@ -10,6 +10,9 @@ import { PlannerAssignments, PlannerBucket, PlannerPlan, PlannerTask } from '@mi
 import { CollectionResponse } from '@microsoft/mgt-element';
 import { ITask } from './task-sources';
 
+const writePlannerDataScopes = ['Tasks.ReadWrite', 'Group.ReadWrite.All'];
+const readPlannerDataScopes = ['Tasks.Read', 'Group.Read.All', 'Group.ReadWrite.All', 'Tasks.ReadWrite'];
+
```

Comment:
Ohhhh interesting. So it looks like this is taken care of here. Our mgt-planner docs will really need to get updated than!